tests/integration/test_auditlogs.py

```diff
@@ -41,6 +67,7 @@
         ("{} '{}' --debug".format(BASE_COMMAND, begin_date_str)),
     ],
 )
-def test_auditlogs_search_command_returns_success_return_code(command, command_runner):
-    return_code, response = command_runner(command)
-    assert return_code == 0
+def test_auditlogs_search_command_returns_success_return_code(
+    runner, integration_test_profile, command, command_runner
```

Comment:
The command runner is already using a temp profile. Why do we need to take another profile fixture here?

Reply:
done.